hbase-server/src/test/java/org/apache/hadoop/hbase/filter/TestFilterList.java

```diff
@@ -169,7 +169,7 @@ private void mpOneTest(Filter filterMPONE) throws Exception {
     assertTrue(filterMPONE.filterRowKey(KeyValueUtil.createFirstOnRow(rowkey)));
     kv = new KeyValue(rowkey, rowkey, Bytes.toBytes(0), Bytes.toBytes(0));
     assertFalse(Filter.ReturnCode.INCLUDE == filterMPONE.filterCell(kv));
-    assertFalse(filterMPONE.filterRow());
+    assertTrue(filterMPONE.filterRow());
```

Comment:
Can you explain this change ?
We are calling filterCell for yyy, which is the prefix. 
Why would that behaviour change ?

Reply:
I reverted this change.